src/app/models/AbstractModel.spec.ts

```diff
@@ -132,6 +132,67 @@ describe("AbstractModel", () => {
     });
   });
 
+  describe("nullEmit", () => {
+    function hasFormDataOnlyAttributes(model: AbstractModel) {
+      return model.hasFormDataOnlyAttributes({ create: true });
+    }
+    const testFile = new File([""], "testFileName.png");
+
+    it("should emit null type in JSON attributes", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ create: true, supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({ id: 1, image: null });
+      expect(Object.keys(model.getJsonAttributes({ create: true }))).toContain("image");
+    });
+
+    it("should not emit file type object in JSON attributes", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ create: true, supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({ id: 1, image: testFile });
+      expect(Object.keys(model.getJsonAttributes({ create: true }))).not.toContain("image");
+    });
+
+    it("should return false if attribute is null", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({image: null});
+      expect(hasFormDataOnlyAttributes(model)).toBeFalse();
+    });
+
+    it("should return true if attribute is a File type object", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({image: testFile});
+      expect(hasFormDataOnlyAttributes(model)).toBeTrue();
+    });
+
+    it("should convert null type values to JSON", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ create: true, supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({ id: 1, image: null });
+      expect(model.toJSON()).toBeNull();
+    });
+
+    it("should not convert image type values to JSON", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ create: true, supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({ id: 1, image: testFile });
+      expect(typeof model.toJSON() === "object").toBeTrue();
```

Comment:
This is an invalid assertion.

```suggestion
      var actual = model.toJSON();
      expect(actual.image).toBeInstanceOf(File);
```